plugin/sampling/strategystore/factory.go

```diff
@@ -34,10 +34,10 @@ type Kind string
 
 const (
 	samplingTypeAdaptive = "adaptive"
-	samplingTypeStatic   = "static"
+	samplingTypeStatic   = "file"
```

Comment:
Based on https://github.com/jaegertracing/jaeger/pull/2966/files#diff-3ee6a5bb916b37c246c98086c1195bb67cbbc19e0166c8c136472b918267c884L31, the value `static` existed before, so we need to support it per backwards compatibility policy (can be removed after 3 months). So I would suggest naming this const samplingTypeFile, and then handling the value "static" in the switch statement in the factory as legacy (ideally with a logged deprecation warning).